Training/configs/training_v1.yaml

```diff
@@ -0,0 +1,285 @@
+Setup:
+    n_tau                :  500
+    start_dataset        :  0
+    end_dataset          :  9999999999999 #std::numeric_limits<Long64_t>::max()
+    n_inner_cells        :  11
+    inner_cell_size      :  0.02
+    n_outer_cells        :  21
+    outer_cell_size      :  0.05
+    n_threads            :  1
+    n_fe_tau             :  43
+    n_pf_el              :  22
+    n_pf_mu              :  23
+    n_pf_chHad           :  27
+    n_pf_nHad            :  7
+    n_pf_gamma           :  23
+    n_ele                :  37
+    n_muon               :  37
+    tau_types            :  4
+    tau_types_names      : {"e","mu","tau","jet"}
+    input_dirs           : {"/eos/user/m/myshched/LawTest_flush10_virt10_v3/"}
+    file_name_pattern    : "\"^.*_(1|2|3|4|5|6|7|8|9|10).root$\""
+    exclude_list         : "\"\""
+    exclude_dir_list     : "\"\""
+    input_spectrum       : "\"/eos/home-m/myshched/ShuffleSpectrum.root\""
+    target_spectrum      : "\"/eos/home-m/myshched/ShuffleSpectrum.root\""
+    n_pt_bins            : 49
+    pt_min               : 20.0
+    pt_max               : 1000.0
+    n_eta_bins           : 4
+    eta_min              : 0.0
+    eta_max              : 2.3
+    weight_thr           : 100000.0
+
+Features_enable :
```

Comment:
names (`Features_enable`, `Features_disable`) and the way it is used is a bit counterintuitive for me.
Maybe (`Features_all`, `Features_enabled`) or (`Features_all`, `Features_disabled`) would be better?